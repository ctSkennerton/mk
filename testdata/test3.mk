# A variable is expanded to decide to run the second rule
deps = one

test3.mk.o: $deps
	secondprocess ab

one:
	process bar
