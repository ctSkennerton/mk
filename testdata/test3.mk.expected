one: process bar
test3.mk.o: secondprocess ab
